macros/PGbasicmacros.pl

```diff
@@ -2112,7 +2112,7 @@ sub general_math_ev3 {
 
 	my $out;
 	if($displayMode eq "HTML_MathJax") {
-		$out = '<span>'.$in_delim.'</span>';
+		$out = '<span><script type="math/tex' . ($mode eq 'display' ? '; mode=display' : '') . '">' . $in . '</script></span>';
```

Comment:
Is there a need for the surrounding `<span>`?  I don't see one myself.

Reply:
It was there before, and I just left it.   It isn't needed as far as I can tell.  I removed it.